Src/AutoFixtureUnitTest/AbstractRecursionIssue/Repro.cs

```diff
@@ -65,7 +65,7 @@ public void Workaround()
                     typeof(ItemBase),
                     typeof(FunkyItem)));
 
-            Assert.DoesNotThrow(() => fixture.Create<FunkyItem>());
+            AssertEx.DoesNotThrow(() => fixture.Create<FunkyItem>());
```

Comment:
>Also I could replace this with the following pair as advised [here](https://github.com/xunit/xunit/issues/188#issuecomment-86584802):
>```
>var ex = Record.Exception(() => sut.Foo());
>Assert.Null(ex);
>```

I think that's better, more straightforward than adding `AssertEx` 😉 What about

```
Assert.Null(Record.Exception(() => sut.Foo()))
```
?